libraries/mapping/carma-map-layers/src/components/LibModal.tsx

```diff
@@ -20,21 +20,26 @@ import LayerTabs from "./LayerTabs";
 import LibItem from "./LibItem";
 import "./input.css";
 import "./modal.css";
-import type { Item, Layer } from "../helper/types";
+import type { Item, Layer, SavedLayerConfig } from "../helper/types";
 import { isEqual } from "lodash";
 const { Search } = Input;
 
 // @ts-expect-error tbd
 const parser = new WMSCapabilities();
 
+type LayerCategories = {
+  Title: string;
+  layers: SavedLayerConfig[];
+}
+
 export interface LibModalProps {
   open: boolean;
   setOpen: (open: boolean) => void;
   setAdditionalLayers: any;
   setThumbnail: any;
   thumbnails: any;
   activeLayers: any[];
-  customCategories?: any[];
+  customCategories?: LayerCategories[];
```

Comment:
should be SavedLayerConfig?

Reply:
resolve if LayerCategories[]; and SavedLayerConfig are the same. one has uppercase Title,
Title: string